### PR TITLE
(Paid) Adds some more donator borgs + fixes a typo

### PR DESCRIPTION
**Pull request by Kmc2000**
### Intent of your Pull Request

https://gyazo.com/fc9628b28c4f58e9ea043ef6792f5bea

Contact FatalEyes if you want to request borg skins.

Brings back some old favourites too.

#### Changelog

:cl: Kmc2000
rscadd: Added new borg skins for donators
/:cl:

### CODE REVIEW

File: yogstation/code/modules/donor/borg_ai_skin_datums.dm [hunk 1]
```diff
@@ -108,6 +108,35 @@
...
+	owner = null
+	module_locked = "Medical"
+
+/datum/borg_skin/drill
+	name = "Drillbot"
+	icon_state = "drillbot"
+	owner = null
+	module_locked = "Miner"
+
+/datum/borg_skin/snailborg
```
Comment (ThatLing) [inline yogstation/code/modules/donor/borg_ai_skin_datums.dm:135]: Yeah, no

**Review by ThatLing — CHANGES_REQUESTED** [review]

**Comment by Kmc2000** [comment]
ready for merge

**Review by Amelia0010 — CHANGES_REQUESTED** [review]
You do not need to do `owner = null`, it inherits that from the parent type.

**Comment by Kmc2000** [comment]
I know Nich.
It's to clearly show which ones are universal

**Comment by Amelia0010** [comment]
🤔

**Comment by Kmc2000** [comment]
Pettiness 100

**Comment by Amelia0010** [comment]
101*

**Pull request merged**